modules/orbitsoftBidAdapter.js

```diff
@@ -50,6 +50,7 @@ export const spec = {
       let requestUrl = utils.getBidIdParameter('requestUrl', bidRequestParams);
       let referrer = utils.getBidIdParameter('ref', bidRequestParams);
       let location = utils.getBidIdParameter('loc', bidRequestParams);
+      let currency = utils.getBidIdParameter('currency', bidRequestParams);
```

Comment:
why put this in your adapter params instead of reading from bidRequest.ortb2 ?

Reply:
replace into ortb2